### DISCUSSION TITLE
Developer pack verification

### DISCUSSION THREAD

**Discussion by shubhamchasing** [open-sauced/app#4167] [Q&A]
Hi team,

As I contribute to OpenSauced, I have to delete old and create new Workspace, Contributor Insight, and Repository Insight to test, debug, and verify the changes I have made are correct and working fine. 

I tried to Verify for the Developer Pack so I do not have to do the manual work so many times but I am getting `Your account is not currently eligible for developer pack`. 

Please help me with this, thank you for your help  🙏

**Comment by nickytonline** [comment]
If you're a student, you should be eligible for this. See https://github.com/orgs/community/discussions/114413

**Reply by shubhamchasing** [reply]
I am not a student although I am a student for life 🤓, jokes apart, hence I am not a student what can be another way? so I can save my time while making contributions to OpenSauced.
